README.md

```diff
@@ -0,0 +1,89 @@
+# DACCS Docker image updater
+
+
+STATUS : Experimental
+
+Automatically updates `birdhouse-deploy` docker images as soon as new tags are pushed.
+
+## Requirements
+
+- hub (via brew)
```

Comment:
Can add, if using docker way, nothing to install locally.   I started to install, then I continue reading the README and found the docker section below ! :(

Reply:
I've moved this section in the CLI section, below Docker section. Makes more sense